query-engine/connector-test-kit-rs/test-configs/pg-postgres13

```diff
@@ -2,5 +2,6 @@
     "connector": "postgres",
     "version": "13",
     "driver_adapter": "pg",
-    "external_test_executor": "default"
+    "external_test_executor": "default",
```

Comment:
I think we could live with only one key for `external_test_executor` that was either `"Napi"` or `"Wasm"`. and not having a key for the engine. Two different runner files encapsulating the configuration of the engine should be fine, and less configuration to deal with.